parser.js

```diff
@@ -44,7 +44,8 @@ const FOXY_COMMANDS = {
   'POCKET': 'POCKET',
   'NPR': 'NPR',
   'GA': 'GA',
-  'FEEDBACK': 'FEEDBACK'
+  'FEEDBACK': 'FEEDBACK',
+  'MUTE': 'MUTE'
```

Comment:
Can you change this to 'shutup' as this will be more clear when we are talking about muting the microphone.  Also, internally, this is talked about as the "shutup" feature.